lib/blueprinter/view_collection.rb

```diff
@@ -35,7 +35,16 @@ def fields_for(view_name)
     end
 
     def transformers(view_name)
-      views[view_name].transformers
+      transformers = gather_transformers_from_included_views(view_name)
+      transformers.presence&.uniq || views[:default].transformers
```

Comment:
`presence` is a Rails specific method. How about this instead?
```suggestion
      transformers && !transformers.empty? ? transformers.uniq : views[:default].transformers
```